src/features/auth/Logister.tsx

```diff
@@ -0,0 +1,180 @@
+import React, {useMemo, useState} from 'react';
+import {View, Text, StyleSheet} from 'react-native';
+import PrimaryButton from '../../components/Buttons/PrimaryButton';
+import InputField from '../../components/InputField/InputField';
+import colors from '../../constants/colors';
+import useToggle from '../../utils/useToggle';
+
+const Logister = () => {
+  const [isSignup, setIsSignup] = useToggle(true);
+  const [email, setEmail] = useState({
+    value: '',
+    isValid: true,
+  });
+  const [password, setPassword] = useState({
+    value: '',
+    isValid: true,
+  });
+  const [rePassword, setRePassword] = useState({
+    value: '',
+    isValid: true,
+  });
+
+  const title = useMemo(() => {
+    return isSignup ? 'Signup' : 'Login';
+  }, [isSignup]);
+
+  const primaryBtnText = useMemo(() => {
+    return isSignup ? 'Signup' : 'Login';
+  }, [isSignup]);
+
+  const secondaryBtnText = useMemo(() => {
+    return isSignup ? 'Login' : 'Signup';
+  }, [isSignup]);
+
+  const validateInput = (type: string, value: string) => {
```

Comment:
remove to utils folder, create validation file and write validation function for each type